### DISCUSSION TITLE
Which social media would you like to see supported next?

### DISCUSSION THREAD

**Discussion by psanders** [TechSquidTV/Shoutify#1] [Polls]
I'm starting this poll knowing it's early for this project, but hoping it could help influence which Social Media gets supported in the future.

Along with the vote, it would be cool if you explained why your answer.

Let's start this party 🎉

**Comment by KyleTryon** [comment]
Nice! Didn't know polls were even a thing. I am _trying_ to remember, there is a commercial social media management tool that was built by someone on Twitter, in fact I am nearly certain that I follow them, but I can not remember who it is or the name of the app lol. They released an open source SDK for interacting with LinkedIn for their own product, if I remember correctly. I'll keep hunting for that and update here if I find it.

**Comment by JobaAdewumi** [comment]
The project seems interesting, is there any way I can contribute or it isn't open yet. Cause I heard about it when you were talking about it on the Twitter space.

**Reply by psanders** [reply]
Hey @JobaAdewumi 

The project is very early but stick around. I feel that soon we will have "good first issues."

**Reply by JobaAdewumi** [reply]
Alright no problem, who knows maybe I'll be the one to open the first issue, let's see how it goes. I'll be watching.

**Comment by KyleTryon** [comment]
+1 to @psanders 

We absolutely are open to contributors but new developers may find it a bit difficult to interact with the repo, only due to how early in development we are. We'd love for you to stick around an keep an eye out. There is also _tons_ of non-dev work that we do actually need right now in the UI/UX space if you happen to be a designer or just have some opinions. 

Feel free to start new discussions for any ideas you have or requests that maybe are not ready to be opened as an issue.

This is our only currently open issue but many more will be coming soon.
https://github.com/TechSquidTV/Shoutify/issues/3

**Reply by JobaAdewumi** [reply]
I could do designs for the product, I'm relatively starting but I could try.
